src/pvrclient-mediaportal.cpp

```diff
@@ -2481,3 +2481,70 @@ PVR_ERROR cPVRClientMediaPortal::GetStreamReadChunkSize(int& chunksize)
   chunksize = 32 * 1024;
   return PVR_ERROR_NO_ERROR;
 }
+
+namespace
+{
+    int GetFileContents(const std::string& url, std::string& content)
+    {
+      content.clear();
+      kodi::vfs::CFile file;
+      if (file.OpenFile(url))
+      {
+        char buffer[1024];
+        while (int bytesRead = file.Read(buffer, 1024))
+          content.append(buffer, bytesRead);
+      }
+
+      return content.length();
+    }
+}
+
+PVR_ERROR cPVRClientMediaPortal::GetRecordingEdl(const kodi::addon::PVRRecording& recording, std::vector<kodi::addon::PVREDLEntry>& edl) // bilkusg
+{
+    cRecording* myrecording = GetRecordingInfo(recording);
+    if (!myrecording)
+      return PVR_ERROR_SERVER_ERROR;
+
+    const char* pvrFilePath = myrecording->FilePath();
+    const std::string& pvrSmbPath = m_tsreader->TranslatePath(pvrFilePath);
```

Comment:
No need for 2 lines here. One will do, also a reference should not be used to store the value so no `&`.

`const std::string pvrSmbPath = m_tsreader->TranslatePath(myrecording->FilePath());`